python/paddle/v2/framework/tests/test_concat_op.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+import numpy as np
+from gradient_checker import GradientChecker, create_op
+from op_test_util import OpTestMeta
+
+
+class TestConcatOp(unittest.TestCase):
+    __metaclass__ = OpTestMeta
+
+    def setUp(self):
+        self.type = "concat"
+        self.inputs = {
+            'X': [
+                np.random.random((16, 32)).astype("float32"), np.random.random(
+                    (8, 32)).astype("float32"), np.random.random(
+                        (2, 32)).astype("float32")
+            ]
+        }
```

Comment:
Maybe short lines as follows:
``` 
x1 =  np.random.random((16, 32)).astype("float32")
x2 = np.random.random((8, 32)).astype("float32")
x3 = np.random.random (2, 32)).astype("float32")
self.inputs = {'X': [x1, x2, x3]}
```

Reply:
Done. 
But the inputs can not be a Python list or `np.array`, and I create a issue #3805 to describe this problem.